src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -223,19 +241,24 @@ void GcsPlacementGroupScheduler::PrepareResources(
 
   const auto lease_client = GetLeaseClientFromNode(node.value());
   const auto node_id = NodeID::FromBinary(node.value()->node_id());
-  RAY_LOG(DEBUG) << "Preparing resource from node " << node_id
-                 << " for a bundle: " << bundle->DebugString();
+  if (RAY_LOG_ENABLED(DEBUG)) {
```

Comment:
The reason why I add this is that there is an operation of string splicing, I suppose this will bring some performance issues, do you think it's unnecessary?

Reply:
You are saying `GenDebugInfoForBundles ` is called when Debug log is not on? Is it true? (also I feel like it doesn't matter much since we anyway printed them before)